Client/WebAuthN/WebAuthnAuthenticateRequest.swift

```diff
@@ -36,8 +36,8 @@ extension WebAuthnAuthenticateRequest: Decodable {
         challenge = try publicKeyDictionary.decode(String.self, forKey: .challenge)
         
         // userPresence is the inverse of userVerification, UP by default is true
-        let userVerifcationString = try publicKeyDictionary.decodeIfPresent(String.self, forKey: .userVerification) ?? "discouraged"
-        userPresence = userVerifcationString == "discouraged"
+        let userVerifcationString = try publicKeyDictionary.decodeIfPresent(String.self, forKey: .userVerification) ?? "required"
```

Comment:
The default UV value if not present is being updated to `discouraged`. See here: https://github.com/w3c/webauthn/pull/1259/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dR2437

Reply:
https://chromium.googlesource.com/chromium/src/+/master/content/browser/webauth/uv_preferred.md

It is still "preferred".. Google Chrome is now throwing a warning in the console for Chrome 77 when a value is not explicitly provided.

Reply:
Looks like the change from `preferred` to `discouraged` was discarded. Good to know. 

```
We have changed Chrome so that, with Chrome 77, any WebAuthn call that doesn't set an explicit value for userVerification will trigger a warning in the Javascript console pointing to this page. This can be silenced by specifying any explicit value. We hope this to point RPs towards considering whether preferred is their desired behaviour.
```

Given the change that was merged here: https://github.com/w3c/webauthn/pull/1140/files - we should set UP to true always.